exercises/01-portfolio-site/src/App.js

```diff
@@ -1,27 +1,53 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import ReactDOM from 'react-dom'
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+  Switch
+} from "react-router-dom";
+
+import About from './pages/About'
+import Contact from './pages/Contact'
+import Project from './pages/Project'
+import Skills from './pages/Skills'
+import Home from './pages/Home'
 
 class App extends Component {
   render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    );
+    return(
+      <Router>
+        <div>
+          <nav>
+            <ul>
+              <li>
+                <Link to="/">Home</Link>
+              </li>
+              <li>
+                <Link to="/About/">About</Link>
+              </li>
+              <li>
+                <Link to="/Contact/">Contact</Link>
+              </li>
+              <li>
+                <Link to="/Project/">Project</Link>
+              </li>
+              <li>
+                <Link to="/Skills/">Skills</Link>
+              </li>
+            </ul>
+          </nav>
+
+          <Switch>
+            <Route path="/About/" component={About} />
+            <Route path="/Contact/" component={Contact} />
+            <Route path="/Project/" exact component={Project} />
+            <Route path="/Skills/" exact component={Skills} />
+            <Route path="/" exact component={Home} />
```

Comment:
I don't see a page not found here yet.  You can follow up this PR  with that work

Reply:
My bad i will do it!